code-challenges.test.js

```diff
@@ -14,33 +14,118 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
+describe ("capital", () => {
+  it ("takes in an array of objects and returns an array with a sentence about each person with their name capitalized", () => {
+      expect(capital(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
+     
+      
+      
+      
+  })
+})
+
+// Good failure
+//   // Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+// Snapshots:   0 total
+// Time:        0.162 s
+
 const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
   { name: "arthur dent", occupation: "a radio employee" }
 ]
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
+'Ford Prefect is a a hitchhiker.',
+  'Zaphod Beeblebrox is a president of the galaxy.',
+  'Arthur Dent is a a radio employee.'
+
 // b) Create the function that makes the test pass.
+// Process
+// Create a function called "capital" that takes in an object of arrays
+// Use the .map method to loop through the objects 
+// Then take the name of each person and split it into an array of characters
+// Then capitalize the first letter of the word using the .uppercase method
+// The join the remaining letters to join the capitalized letter to form our new words
+// Use string interpolation to return the desired statement
+
+const capital= (people) => {
+  people.map(person => {
```

Comment:
Your pseudo code is pretty good, but this function is returning undefined because you are missing a return. Your .map has a return, but your function itself does not!